python/mxnet/optimizer.py

```diff
@@ -892,18 +893,35 @@ def update(self, index, weight, grad, state):
         weight[:] += mom
 
 @register
-class NAG(SGD):
+@register
```

Comment:
Two "@register" lead to warning "UserWarning: WARNING: New optimizer mxnet.optimizer.NAG is overriding existing optimizer mxnet.optimizer.NAG Optimizer.opt_registry[name].__name__))" while importing mxnet.

Reply:
Thanks for looking into this. Do you mind posting a PR to remove the extra @register?

Reply:
sure.